app/client/src/components/pages/Actions.js

```diff
@@ -749,19 +760,18 @@ function Actions() {
                                       )}
 
                                     <p css={paragraphContentStyles}>
-                                      {graphic && (
-                                        <ViewOnMapButton
-                                          feature={{
-                                            attributes: {
-                                              assessmentunitidentifier: auId,
-                                              organizationid: orgId,
-                                              fieldName: 'hmw-extra-content',
-                                            },
-                                          }}
-                                          layers={[mapLayer.layer]}
-                                          fieldName="hmw-extra-content"
-                                        />
-                                      )}
+                                      <ViewOnMapButton
+                                        feature={{
+                                          attributes: {
+                                            assessmentunitidentifier: auId,
+                                            organizationid: orgId,
+                                            fieldName: 'hmw-extra-content',
+                                          },
+                                        }}
+                                        layers={[mapLayer.layer]}
+                                        fieldName="hmw-extra-content"
+                                        disabled={graphic ? false : true}
+                                      />
```

Comment:
For consistency with the above suggestion, we'd want to check for the status of the `mapLayer` here, too:
```suggestion
                                      {mapLayer.status === 'success' && (
                                        <ViewOnMapButton
                                          feature={{
                                            attributes: {
                                              assessmentunitidentifier: auId,
                                              organizationid: orgId,
                                              fieldName: 'hmw-extra-content',
                                            },
                                          }}
                                          layers={[mapLayer.layer]}
                                          fieldName="hmw-extra-content"
                                          disabled={graphic ? false : true}
                                        />
                                      )}
```